### PR TITLE
Created Telephone Numbers md  formatting part

### PR DESCRIPTION
**Pull request by RobTobias123**
Initial draft of formatting information. To be continued with an Accessibility guidelines section

### CODE REVIEW

File: docs/foundations/telephone-numbers.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+To aid memorability, the patterns we use for freephone, non-geographical or emergency numbers are as follows:
+
+| Format | Usage
+| :--- | :--- 
+| 0800&nbsp;XXX&nbsp;XXX<br>0800&nbsp;XXX&nbsp;XXXX | Freephone |
+| 0333&nbsp;XXX&nbsp;XXXX | Non-geographic |
+| 105 | Electrical emergency |
+| 0800&nbsp;111&nbsp;999 | Gas emergency |
+
+<br>
```
Comment (dbenmore) [inline docs/foundations/telephone-numbers.md:23]: `<br>` tags are forbidden

**Comment by andij** [comment]
Let's find a more suitable location for this. It's not one of the design system foundations. 

My suggestion would be to include this information in: https://nucleus.design/docs/patterns/asking-for-information

**Comment by PtitBen** [comment]
I think this has two sides, the one Rob is looking at at the moment is not asking for information but rather playing back information and make it more accessible/usable.

To Dan's point in Teams, I think Rob has been looking at `tel:` and `sms:` links too

**Comment by PtitBen** [comment]
Can we cover for flexible formatting?
If the system is consumed by a non-UK-based team, they might need to use a different set of rules.

Example: 
Belgium has the following formatting:

- Big Cities: 01 xxx xxx
- Smaller towns grouped in areas: 012 xxx xxx
- Mobile numbers: 04xx xxx xxx
- Text numbers are 4 digits

**Comment by andij** [comment]
Ah, fair enough. Does this raise the potential need to introduce a foundation for 'Content'?

We could use it for telephone number formatting, (interactive and non interactive) email addresses and any other special type of content.

**Review by PtitBen — APPROVED** [review]

**Pull request merged**